README.md

```diff
@@ -90,10 +90,13 @@ The [Helm chart](https://artifacthub.io/packages/helm/metrics-server/metrics-ser
 
 ## Security context
 
-Metrics Server requires the `CAP_NET_BIND_SERVICE` capability in order to bind to a privileged ports as non-root.
-If you are running Metrics Server in an environment that uses PSPs or other mechanisms to restrict pod capabilities, ensure that Metrics Server is allowed
-to use this capability.
-This applies even if you use the `--secure-port` flag to change the port that Metrics Server binds to to a non-privileged port.
+Metrics Server may require the `CAP_NET_BIND_SERVICE` capability, in order to bind to a privileged port as non-root.
+As such, Kubernetes versions [prior to v1.23.0](https://github.com/kubernetes/kubernetes/commit/3d620192d95dbacf5ad9c18755576b15047a8d10) require this capability.
```

Comment:
a small doubt,  [prior to v1.23.0](https://github.com/kubernetes/kubernetes/commit/3d620192d95dbacf5ad9c18755576b15047a8d10),
Is the change `CAP_NET_BIND_SERVICE` here related to this manifests?

Reply:
@yangjunmyfm192085 my understanding is that `CAP_NET_BIND_SERVICE` was directly necessary due to `setcap cap_net_bind_service=+ep metrics-server` being used [in the docker image](https://github.com/kubernetes-sigs/metrics-server/commit/d3265cf376031455926bc86fcfae4a7a28226e9e). I've assumed that the reason for this was to enable or ease binding to a privileged port, behavior that I believe was changed in the " [prior to v1.23.0](https://github.com/kubernetes/kubernetes/commit/3d620192d95dbacf5ad9c18755576b15047a8d10) " link.

It would be good to get somebody to validate that my assumptions are correct.

Reply:
I think I understand what you mean. But the problem is that the metrics-server release and the kubernetes release are not binding to publish. Is the description here necessary?

Reply:
Even without `setcap` in the Dockerfile, if Metrics Server binds to a privileged port, then it'll need to be privileged. Depending on how a given user deployed their cluster, they might have the option to specify the version of Metrics Server that is used. The newer versions of Metrics Server without `setcap` won't escalate automatically, which has the potential to break the use of those versions with a cluster that configures Metrics Server to use a privileged port.